src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py

```diff
@@ -2206,6 +2206,10 @@ def test_keyvault_softdelete(self, resource_group, key_vault, key_vault2):
         self.cmd('keyvault key create --vault-name {kv} -n key2 -p software',
                  checks=self.check('attributes.enabled', True))
 
+        # test key get-policy-template
+        self.cmd('keyvault key get-policy-template',
+                 checks=self.check('length(@)', 2))
```

Comment:
The `get-policy-template` result is a `json` not a `list`, why check the `length(@)`?

Reply:
It aims to make sure the command works well and have results return.

Reply:
How about add some meaningful checks... Even checking the result not `None` also makes sense.

Reply:
This check just means not none.